README.md

```diff
@@ -84,10 +84,11 @@ Our code development features a mix of upstream OpenStack work and custom ESI co
 
 ## Contact and Contribution
 
-You can contact the ESI development team on the Freenode `#moc` IRC channel.
+You can contact the ESI development team on the [OFTC][oftc] `#moc` IRC channel.
```

Comment:
Why is this commit included in your PR?

Reply:
I am bad at git
I was trying to fix a merge conflict but I guess I messed it up